### ISSUE TITLE
Error in jwt:secret command on Lumen

### ISSUE THREAD

**Issue by cdarken** [tymondesigns/jwt-auth#1280]
Sorry if it's too soon to post this bug, as it's based on code found on develop branch, but when running the command that generates the secret, I get this error:

```
Call to undefined method Laravel\Lumen\Application::environmentFilePath()
```

**Comment by paulschwoerer** [comment]
I had the same issue with the develop branch and replaced the following (around line number 60) in `JWTGenerateSecretCommand`:
``` 
$this->laravel->environmentFilePath()
```
with
```
$this->laravel->basePath() . DIRECTORY_SEPARATOR . '.env'
```
This will work fine for the default .env file in both Lumen and Laravel.
Maybe @tymondesigns, you are able to add a Lumen speacial case. Thanks :)

**Comment by tymondesigns** [comment]
Thanks for catching this, should be fixed as per 234648884b78e987686597964a6496cf78780cb5
